Simplenote/Classes/NSRegularExpression+Simplenote.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+
+// MARK: - NSRegularExpression
+//
+extension NSRegularExpression {
+
+    /// Matches Checklists at the beginning of each line
+    ///
+    @objc
+    static let regexForChecklists: NSRegularExpression = {
+        try! NSRegularExpression(pattern: "^\\s*(-[ \t]+\\[[xX\\s]\\])", options: .anchorsMatchLines)
```

Comment:
In the Electron app we use this…

`^(\s*)(-[ \t]+\[[xX\s]?\])(.+)`

Looks like the difference is that in the Electron app we treat ` - [] ` as an unchecked todo. I'm not sure this is right from a purity standpoint though it probably is easier for people who don't expect that they need the space inside the box.

Just pointing out the difference is all

Reply:
Thanks for mentioning this @dmsnell, I'll get the regex updated so that the behavior is the same!